src/app.js

```diff
@@ -39,9 +39,9 @@ app.use('/.well-known', wellKnownRouter);
 app.use('/public', publicRouter);
 
 // Protected Routes
-app.use('/', backendAuthorization, indexRouter);
-app.use('/fhir', backendAuthorization, fhirRouter);
-app.use('/servers', backendAuthorization, serversRouter);
-app.use('/notif', subscriptionAuthorization, subscriptionsRouter);
+app.use('/api',backendAuthorization, indexRouter);
```

Comment:
Can you be sure to update the wiki with these new endpoints please

Reply:
i decided to revert some of the `/api` endpoints, since they didn't make sense, but I'll update the index one.